Aspire Budgeting/Views/CardView/BaseCardView.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  BaseCardView.swift
+//  Aspire Budgeting
+//
+
+import SwiftUI
+
+struct BaseCardView<Content: View>: View {
+  var minY: CGFloat = 0
+  var curY: CGFloat = 0
+
+  private let cornerRadius: CGFloat = 24
+  private let height: CGFloat = 163
+
+  private let shadowRadius: CGFloat = 14
+  private let shadowYOffset: CGFloat = 4
+
+  private var baseColor: Color
+  private let content: Content
+
+//  private var gradient: LinearGradient {
+//    Color.fondGradientFrom(startColor: colorInfo.gradientStartColor,
+//                           endColor: colorInfo.gradientEndColor)
+//  }
+
+  private var offsetY: CGFloat {
+    curY < minY ? minY - curY : 0
+  }
+
+  init(minY: CGFloat = 0,
+       curY: CGFloat = 0,
+       baseColor: Color,
+       @ViewBuilder content: () -> Content) {
+    self.minY = minY
+    self.curY = curY
+    self.baseColor = baseColor
+    self.content = content()
+  }
+
+  var body: some View {
+    ZStack(alignment: .top) {
+      containerBox
+      content
+    }
+    .offset(y: offsetY)
+  }
+}
+
+extension BaseCardView {
+  private var containerBox: some View {
+    Rectangle()
+      .fill(baseColor)
+      .cornerRadius(cornerRadius)
+      .frame(height: height)
+  }
+}
+
+// MARK: - Previews
+struct CardView_Previews: PreviewProvider {
+  static var previews: some View {
+    Group {
+      Group {
+        BaseCardView<Color>(baseColor: .materialBlue800) {Color.materialBlue800}
+
+        BaseCardView<Color>(baseColor: .materialTeal800) {Color.materialTeal800}
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/5d9f74b3b35e3fb9746b1333ffd07f18323321a5/Aspire Budgeting/Views/CardView/BaseCardView.swift#L65"></span>Closure expressions should have a single space inside each brace.<br />
<code>closure_spacing</code> <code>BaseCardView.swift:65</code></td>
    </tr>
  </tbody>
</table>